src/components/inputs/TextInput.js

```diff
@@ -289,24 +331,27 @@ export default class TextInput extends BaseInput {
   renderCharCounter() {
     const {focused} = this.state;
     const {maxLength, showCharacterCounter} = this.props;
+    
     if (maxLength && showCharacterCounter) {
       const counter = this.getCharCount();
       const color = this.isCounterLimit() && focused ? DEFAULT_COLOR_BY_STATE.error : DEFAULT_COLOR_BY_STATE.default;
       return (
         <Text
-          style={[{color}, this.styles.charCounter]}
+          style={[{color}, this.styles.bottomLabel, this.styles.label]}
         >
           {counter} / {maxLength}
         </Text>
       );
     }
   }
 
-  renderError() {
-    const {enableErrors, error} = this.props;
-    if (enableErrors) {
+  renderError(visible) {
```

Comment:
why do we need to pass visible to `renderError` if the value comes from the props. 
shouldn't  we extract the value directly from the props inside the method to avoid unnecessary params ?